stratum/hal/lib/barefoot/bf_switch.cc

```diff
@@ -277,9 +278,10 @@ ::util::Status BFSwitch::RetrieveValue(uint64 node_id,
         break;
       }
       default:
-        status =
-            MAKE_ERROR(ERR_UNIMPLEMENTED)
-            << "Request type "
+        status = MAKE_ERROR(ERR_UNIMPLEMENTED).without_logging()
+                 << "Not supported yet.";
+        VLOG(1)
+            << "DataRequest field "
```

Comment:
Since we decided that everyone must implement the full switch interface, is this change still valid?

If there is a unhandled type, should not be logged as ERROR and fixed (i.e with a dummy value like bcm_switch)?

Reply:
This is a good point. I'll revert this and make sure the error gets logged in all the switches